test/functional/page_objects/getting_started_page.js

```diff
@@ -1,13 +1,30 @@
-export function GettingStartedPageProvider({ getService }) {
+export function GettingStartedPageProvider({ getService, getPageObjects }) {
 
   const log = getService('log');
   const testSubjects = getService('testSubjects');
 
+  const PageObjects = getPageObjects(['common']);
+
   class GettingStartedPage {
```

Comment:
I'm not sure TBH. I was following the pattern already established with other page objects. If we want to change this pattern I suggest we open a new issue for doing it across the board for all page objects and then fix it there.